src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -204,74 +211,85 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             }
 
             // Build our query
-            var request = BuildQuery(searchTerm, query, ApiEndpoint);
 
-            // Getting results & Store content
-            var results = await QueryExec(request);
-
-            try
+            for (var page = 0; page <= MaxSearchPageLimit; page++)
             {
-                // Deserialize our Json Response
-                var xthorResponse = JsonConvert.DeserializeObject<XthorResponse>(results);
+                var request = BuildQuery(searchTerm, query, ApiEndpoint);
 
-                // Check Tracker's State
-                CheckApiState(xthorResponse.error);
+                // Add option page at the end of the request variable
+                request = request + "&page=" + page;
```

Comment:
change `BuildQuery` function to include the page and do all the logic to generate the URL in there.